src/templates/home.js

```diff
@@ -18,13 +18,17 @@ import React, {Component} from 'react';
 import TitleAndMetaTags from 'components/TitleAndMetaTags';
 import {colors, media, sharedStyles} from 'theme';
 import createOgUrl from 'utils/createOgUrl';
+import loadScript from 'utils/loadScript';
+import {babelURL} from 'site-constants';
 
 class Home extends Component {
   componentDidMount() {
-    mountCodeExample('helloExample', HELLO_COMPONENT);
-    mountCodeExample('timerExample', TIMER_COMPONENT);
-    mountCodeExample('todoExample', TODO_COMPONENT);
-    mountCodeExample('markdownExample', MARKDOWN_COMPONENT);
+    loadScript(babelURL).then(() => {
+      mountCodeExample('helloExample', HELLO_COMPONENT);
+      mountCodeExample('timerExample', TIMER_COMPONENT);
+      mountCodeExample('todoExample', TODO_COMPONENT);
+      mountCodeExample('markdownExample', MARKDOWN_COMPONENT);
+    });
```

Comment:
What are your thoughts about showing a loading placeholder in the 4 divs in question until Babel has finished loading?

Reply:
It's a good idea, I can see that it is flashing once it loads. To fix this should I style the divs at content/index.md?

Reply:
Probably not. (We actually need to clean up `index.md`, see #97)

I would try maybe temporarily rendering a placeholder into the container from the Home template.

Reply:
So you mean that I should use `ReactDOM.render`  like `mountCodeExample` does?

Reply:
Probably 😄